File: 1295. Find Numbers with Even Number of Digits/Find_Numbers_with_Even_Number_of_Digits.py
class Solution(object):

    def findLen(self,nums):
        k = nums // 10
        len = 1
        while k != 0:
            len += 1
            k = k // 10
        return len

    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for i in nums:
            if self.findLen(i) % 2 == 0:
                count += 1
        return count


nums = [12,345,2,6,7896]
print(Solution().findNumbers(nums))

File: 1295. Find Numbers with Even Number of Digits/Find_Numbers_with_Even_Number_of_Digits.java
class Find_Numbers_with_Even_Number_of_Digits {


    public static int findLenNumber (int nums) {
        int k = nums / 10;
        int len = 1;
        while (k != 0 ) {
            len ++;
            k = k / 10;
        }
        return len;
    }

    public static int findNumbers(int[] nums) {
        int count = 0;
        for (int i : nums) {
            int len =findLenNumber(i);
            if (len % 2==0) {
                count ++;
            }
        }
        return count;
    }

    public static void main(String[] args) {
       int[] nums = {12,345,2,6,7896};
       System.out.print(findNumbers(nums));
    }

}